[2024-08-09T19:43:39.022+0200] {processor.py:161} INFO - Started process (PID=17827) to work on /home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-09T19:43:39.023+0200] {processor.py:830} INFO - Processing file /home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-08-09T19:43:39.024+0200] {logging_mixin.py:188} INFO - [2024-08-09T19:43:39.023+0200] {dagbag.py:545} INFO - Filling up the DagBag from /home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-09T19:43:39.025+0200] {logging_mixin.py:188} INFO - [2024-08-09T19:43:39.025+0200] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-08-09T19:43:39.025+0200] {logging_mixin.py:188} INFO - [2024-08-09T19:43:39.025+0200] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-08-09T19:43:39.025+0200] {processor.py:842} WARNING - No viable dags retrieved from /home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-09T19:43:39.031+0200] {processor.py:183} INFO - Processing /home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.012 seconds
[2024-08-09T19:44:40.604+0200] {processor.py:161} INFO - Started process (PID=18113) to work on /home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-09T19:44:40.605+0200] {processor.py:830} INFO - Processing file /home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-08-09T19:44:40.605+0200] {logging_mixin.py:188} INFO - [2024-08-09T19:44:40.605+0200] {dagbag.py:545} INFO - Filling up the DagBag from /home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-09T19:44:40.607+0200] {logging_mixin.py:188} INFO - [2024-08-09T19:44:40.607+0200] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-08-09T19:44:40.607+0200] {logging_mixin.py:188} INFO - [2024-08-09T19:44:40.607+0200] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-08-09T19:44:40.607+0200] {processor.py:842} WARNING - No viable dags retrieved from /home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-09T19:44:40.613+0200] {processor.py:183} INFO - Processing /home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.012 seconds
[2024-08-09T19:46:15.706+0200] {processor.py:161} INFO - Started process (PID=18585) to work on /home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-09T19:46:15.707+0200] {processor.py:830} INFO - Processing file /home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-08-09T19:46:15.708+0200] {logging_mixin.py:188} INFO - [2024-08-09T19:46:15.707+0200] {dagbag.py:545} INFO - Filling up the DagBag from /home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-09T19:46:15.709+0200] {logging_mixin.py:188} INFO - [2024-08-09T19:46:15.709+0200] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-08-09T19:46:15.709+0200] {logging_mixin.py:188} INFO - [2024-08-09T19:46:15.709+0200] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-08-09T19:46:15.710+0200] {processor.py:842} WARNING - No viable dags retrieved from /home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-09T19:46:15.715+0200] {processor.py:183} INFO - Processing /home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.012 seconds
[2024-08-09T19:47:22.925+0200] {processor.py:161} INFO - Started process (PID=18889) to work on /home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-09T19:47:22.926+0200] {processor.py:830} INFO - Processing file /home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-08-09T19:47:22.926+0200] {logging_mixin.py:188} INFO - [2024-08-09T19:47:22.926+0200] {dagbag.py:545} INFO - Filling up the DagBag from /home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-09T19:47:22.928+0200] {logging_mixin.py:188} INFO - [2024-08-09T19:47:22.928+0200] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-08-09T19:47:22.928+0200] {logging_mixin.py:188} INFO - [2024-08-09T19:47:22.928+0200] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-08-09T19:47:22.928+0200] {processor.py:842} WARNING - No viable dags retrieved from /home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-09T19:47:22.933+0200] {processor.py:183} INFO - Processing /home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.011 seconds
[2024-08-09T19:48:38.671+0200] {processor.py:161} INFO - Started process (PID=19116) to work on /home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-09T19:48:38.672+0200] {processor.py:830} INFO - Processing file /home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-08-09T19:48:38.673+0200] {logging_mixin.py:188} INFO - [2024-08-09T19:48:38.673+0200] {dagbag.py:545} INFO - Filling up the DagBag from /home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-09T19:48:38.674+0200] {logging_mixin.py:188} INFO - [2024-08-09T19:48:38.674+0200] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-08-09T19:48:38.675+0200] {logging_mixin.py:188} INFO - [2024-08-09T19:48:38.675+0200] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-08-09T19:48:38.675+0200] {processor.py:842} WARNING - No viable dags retrieved from /home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-09T19:48:38.680+0200] {processor.py:183} INFO - Processing /home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.012 seconds
[2024-08-09T19:49:55.090+0200] {processor.py:161} INFO - Started process (PID=19307) to work on /home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-09T19:49:55.091+0200] {processor.py:830} INFO - Processing file /home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-08-09T19:49:55.092+0200] {logging_mixin.py:188} INFO - [2024-08-09T19:49:55.092+0200] {dagbag.py:545} INFO - Filling up the DagBag from /home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-09T19:49:55.093+0200] {logging_mixin.py:188} INFO - [2024-08-09T19:49:55.093+0200] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-08-09T19:49:55.093+0200] {logging_mixin.py:188} INFO - [2024-08-09T19:49:55.093+0200] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-08-09T19:49:55.094+0200] {processor.py:842} WARNING - No viable dags retrieved from /home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-09T19:49:55.099+0200] {processor.py:183} INFO - Processing /home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.011 seconds
[2024-08-09T19:51:12.569+0200] {processor.py:161} INFO - Started process (PID=19450) to work on /home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-09T19:51:12.572+0200] {processor.py:830} INFO - Processing file /home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-08-09T19:51:12.575+0200] {logging_mixin.py:188} INFO - [2024-08-09T19:51:12.574+0200] {dagbag.py:545} INFO - Filling up the DagBag from /home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-09T19:51:12.581+0200] {logging_mixin.py:188} INFO - [2024-08-09T19:51:12.580+0200] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-08-09T19:51:12.582+0200] {logging_mixin.py:188} INFO - [2024-08-09T19:51:12.581+0200] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-08-09T19:51:12.582+0200] {processor.py:842} WARNING - No viable dags retrieved from /home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-09T19:51:12.594+0200] {processor.py:183} INFO - Processing /home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.041 seconds
[2024-08-09T19:53:08.142+0200] {processor.py:161} INFO - Started process (PID=20077) to work on /home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-09T19:53:08.143+0200] {processor.py:830} INFO - Processing file /home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-08-09T19:53:08.144+0200] {logging_mixin.py:188} INFO - [2024-08-09T19:53:08.143+0200] {dagbag.py:545} INFO - Filling up the DagBag from /home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-09T19:53:08.145+0200] {logging_mixin.py:188} INFO - [2024-08-09T19:53:08.145+0200] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-08-09T19:53:08.145+0200] {logging_mixin.py:188} INFO - [2024-08-09T19:53:08.145+0200] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-08-09T19:53:08.145+0200] {processor.py:842} WARNING - No viable dags retrieved from /home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-09T19:53:08.151+0200] {processor.py:183} INFO - Processing /home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.012 seconds
[2024-08-09T19:54:28.111+0200] {processor.py:161} INFO - Started process (PID=20170) to work on /home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-09T19:54:28.111+0200] {processor.py:830} INFO - Processing file /home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-08-09T19:54:28.112+0200] {logging_mixin.py:188} INFO - [2024-08-09T19:54:28.112+0200] {dagbag.py:545} INFO - Filling up the DagBag from /home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-09T19:54:28.114+0200] {logging_mixin.py:188} INFO - [2024-08-09T19:54:28.113+0200] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-08-09T19:54:28.114+0200] {logging_mixin.py:188} INFO - [2024-08-09T19:54:28.114+0200] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-08-09T19:54:28.114+0200] {processor.py:842} WARNING - No viable dags retrieved from /home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-09T19:54:28.119+0200] {processor.py:183} INFO - Processing /home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.012 seconds
[2024-08-09T19:55:42.939+0200] {processor.py:161} INFO - Started process (PID=20303) to work on /home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-09T19:55:42.939+0200] {processor.py:830} INFO - Processing file /home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-08-09T19:55:42.940+0200] {logging_mixin.py:188} INFO - [2024-08-09T19:55:42.940+0200] {dagbag.py:545} INFO - Filling up the DagBag from /home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-09T19:55:42.941+0200] {logging_mixin.py:188} INFO - [2024-08-09T19:55:42.941+0200] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-08-09T19:55:42.942+0200] {logging_mixin.py:188} INFO - [2024-08-09T19:55:42.942+0200] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-08-09T19:55:42.942+0200] {processor.py:842} WARNING - No viable dags retrieved from /home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-09T19:55:42.947+0200] {processor.py:183} INFO - Processing /home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.011 seconds
[2024-08-09T19:56:54.608+0200] {processor.py:161} INFO - Started process (PID=20384) to work on /home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-09T19:56:54.609+0200] {processor.py:830} INFO - Processing file /home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-08-09T19:56:54.609+0200] {logging_mixin.py:188} INFO - [2024-08-09T19:56:54.609+0200] {dagbag.py:545} INFO - Filling up the DagBag from /home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-09T19:56:54.611+0200] {logging_mixin.py:188} INFO - [2024-08-09T19:56:54.611+0200] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-08-09T19:56:54.611+0200] {logging_mixin.py:188} INFO - [2024-08-09T19:56:54.611+0200] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-08-09T19:56:54.611+0200] {processor.py:842} WARNING - No viable dags retrieved from /home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-09T19:56:54.616+0200] {processor.py:183} INFO - Processing /home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.011 seconds
[2024-08-09T19:58:12.206+0200] {processor.py:161} INFO - Started process (PID=20544) to work on /home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-09T19:58:12.215+0200] {processor.py:830} INFO - Processing file /home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-08-09T19:58:12.216+0200] {logging_mixin.py:188} INFO - [2024-08-09T19:58:12.216+0200] {dagbag.py:545} INFO - Filling up the DagBag from /home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-09T19:58:12.218+0200] {logging_mixin.py:188} INFO - [2024-08-09T19:58:12.218+0200] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-08-09T19:58:12.218+0200] {logging_mixin.py:188} INFO - [2024-08-09T19:58:12.218+0200] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-08-09T19:58:12.219+0200] {processor.py:842} WARNING - No viable dags retrieved from /home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-09T19:58:12.225+0200] {processor.py:183} INFO - Processing /home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.023 seconds
[2024-08-09T19:59:38.167+0200] {processor.py:161} INFO - Started process (PID=20713) to work on /home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-09T19:59:38.186+0200] {processor.py:830} INFO - Processing file /home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-08-09T19:59:38.187+0200] {logging_mixin.py:188} INFO - [2024-08-09T19:59:38.187+0200] {dagbag.py:545} INFO - Filling up the DagBag from /home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-09T19:59:38.189+0200] {logging_mixin.py:188} INFO - [2024-08-09T19:59:38.189+0200] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-08-09T19:59:38.189+0200] {logging_mixin.py:188} INFO - [2024-08-09T19:59:38.189+0200] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-08-09T19:59:38.189+0200] {processor.py:842} WARNING - No viable dags retrieved from /home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-09T19:59:38.195+0200] {processor.py:183} INFO - Processing /home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.031 seconds
[2024-08-09T20:00:53.420+0200] {processor.py:161} INFO - Started process (PID=20817) to work on /home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-09T20:00:53.420+0200] {processor.py:830} INFO - Processing file /home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-08-09T20:00:53.421+0200] {logging_mixin.py:188} INFO - [2024-08-09T20:00:53.421+0200] {dagbag.py:545} INFO - Filling up the DagBag from /home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-09T20:00:53.423+0200] {logging_mixin.py:188} INFO - [2024-08-09T20:00:53.422+0200] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-08-09T20:00:53.423+0200] {logging_mixin.py:188} INFO - [2024-08-09T20:00:53.423+0200] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-08-09T20:00:53.423+0200] {processor.py:842} WARNING - No viable dags retrieved from /home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-09T20:00:53.428+0200] {processor.py:183} INFO - Processing /home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.012 seconds
[2024-08-09T20:02:07.851+0200] {processor.py:161} INFO - Started process (PID=20979) to work on /home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-09T20:02:07.852+0200] {processor.py:830} INFO - Processing file /home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-08-09T20:02:07.852+0200] {logging_mixin.py:188} INFO - [2024-08-09T20:02:07.852+0200] {dagbag.py:545} INFO - Filling up the DagBag from /home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-09T20:02:07.854+0200] {logging_mixin.py:188} INFO - [2024-08-09T20:02:07.854+0200] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-08-09T20:02:07.854+0200] {logging_mixin.py:188} INFO - [2024-08-09T20:02:07.854+0200] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-08-09T20:02:07.854+0200] {processor.py:842} WARNING - No viable dags retrieved from /home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-09T20:02:07.859+0200] {processor.py:183} INFO - Processing /home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.011 seconds
[2024-08-09T20:03:34.418+0200] {processor.py:161} INFO - Started process (PID=21105) to work on /home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-09T20:03:34.419+0200] {processor.py:830} INFO - Processing file /home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-08-09T20:03:34.419+0200] {logging_mixin.py:188} INFO - [2024-08-09T20:03:34.419+0200] {dagbag.py:545} INFO - Filling up the DagBag from /home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-09T20:03:34.421+0200] {logging_mixin.py:188} INFO - [2024-08-09T20:03:34.421+0200] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-08-09T20:03:34.421+0200] {logging_mixin.py:188} INFO - [2024-08-09T20:03:34.421+0200] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-08-09T20:03:34.421+0200] {processor.py:842} WARNING - No viable dags retrieved from /home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-09T20:03:34.427+0200] {processor.py:183} INFO - Processing /home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.012 seconds
[2024-08-09T20:04:52.327+0200] {processor.py:161} INFO - Started process (PID=21249) to work on /home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-09T20:04:52.328+0200] {processor.py:830} INFO - Processing file /home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-08-09T20:04:52.330+0200] {logging_mixin.py:188} INFO - [2024-08-09T20:04:52.329+0200] {dagbag.py:545} INFO - Filling up the DagBag from /home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-09T20:04:52.333+0200] {logging_mixin.py:188} INFO - [2024-08-09T20:04:52.332+0200] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-08-09T20:04:52.334+0200] {logging_mixin.py:188} INFO - [2024-08-09T20:04:52.333+0200] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-08-09T20:04:52.334+0200] {processor.py:842} WARNING - No viable dags retrieved from /home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-09T20:04:52.342+0200] {processor.py:183} INFO - Processing /home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.022 seconds
[2024-08-09T20:06:16.435+0200] {processor.py:161} INFO - Started process (PID=21407) to work on /home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-09T20:06:16.455+0200] {processor.py:830} INFO - Processing file /home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-08-09T20:06:16.456+0200] {logging_mixin.py:188} INFO - [2024-08-09T20:06:16.456+0200] {dagbag.py:545} INFO - Filling up the DagBag from /home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-09T20:06:16.459+0200] {logging_mixin.py:188} INFO - [2024-08-09T20:06:16.458+0200] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-08-09T20:06:16.459+0200] {logging_mixin.py:188} INFO - [2024-08-09T20:06:16.459+0200] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-08-09T20:06:16.459+0200] {processor.py:842} WARNING - No viable dags retrieved from /home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-09T20:06:16.465+0200] {processor.py:183} INFO - Processing /home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.033 seconds
[2024-08-09T20:07:37.241+0200] {processor.py:161} INFO - Started process (PID=21598) to work on /home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-09T20:07:37.242+0200] {processor.py:830} INFO - Processing file /home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-08-09T20:07:37.243+0200] {logging_mixin.py:188} INFO - [2024-08-09T20:07:37.242+0200] {dagbag.py:545} INFO - Filling up the DagBag from /home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-09T20:07:37.244+0200] {logging_mixin.py:188} INFO - [2024-08-09T20:07:37.244+0200] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-08-09T20:07:37.244+0200] {logging_mixin.py:188} INFO - [2024-08-09T20:07:37.244+0200] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-08-09T20:07:37.244+0200] {processor.py:842} WARNING - No viable dags retrieved from /home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-09T20:07:37.249+0200] {processor.py:183} INFO - Processing /home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.011 seconds
[2024-08-09T20:08:52.022+0200] {processor.py:161} INFO - Started process (PID=21712) to work on /home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-09T20:08:52.023+0200] {processor.py:830} INFO - Processing file /home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-08-09T20:08:52.024+0200] {logging_mixin.py:188} INFO - [2024-08-09T20:08:52.023+0200] {dagbag.py:545} INFO - Filling up the DagBag from /home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-09T20:08:52.025+0200] {logging_mixin.py:188} INFO - [2024-08-09T20:08:52.025+0200] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-08-09T20:08:52.025+0200] {logging_mixin.py:188} INFO - [2024-08-09T20:08:52.025+0200] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-08-09T20:08:52.025+0200] {processor.py:842} WARNING - No viable dags retrieved from /home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-09T20:08:52.031+0200] {processor.py:183} INFO - Processing /home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.012 seconds
[2024-08-09T20:10:08.708+0200] {processor.py:161} INFO - Started process (PID=21809) to work on /home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-09T20:10:08.708+0200] {processor.py:830} INFO - Processing file /home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-08-09T20:10:08.709+0200] {logging_mixin.py:188} INFO - [2024-08-09T20:10:08.709+0200] {dagbag.py:545} INFO - Filling up the DagBag from /home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-09T20:10:08.711+0200] {logging_mixin.py:188} INFO - [2024-08-09T20:10:08.710+0200] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-08-09T20:10:08.711+0200] {logging_mixin.py:188} INFO - [2024-08-09T20:10:08.711+0200] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-08-09T20:10:08.711+0200] {processor.py:842} WARNING - No viable dags retrieved from /home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-09T20:10:08.716+0200] {processor.py:183} INFO - Processing /home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.011 seconds
[2024-08-09T20:11:24.157+0200] {processor.py:161} INFO - Started process (PID=21893) to work on /home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-09T20:11:24.158+0200] {processor.py:830} INFO - Processing file /home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-08-09T20:11:24.159+0200] {logging_mixin.py:188} INFO - [2024-08-09T20:11:24.158+0200] {dagbag.py:545} INFO - Filling up the DagBag from /home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-09T20:11:24.160+0200] {logging_mixin.py:188} INFO - [2024-08-09T20:11:24.160+0200] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-08-09T20:11:24.160+0200] {logging_mixin.py:188} INFO - [2024-08-09T20:11:24.160+0200] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-08-09T20:11:24.161+0200] {processor.py:842} WARNING - No viable dags retrieved from /home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-09T20:11:24.166+0200] {processor.py:183} INFO - Processing /home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.012 seconds
[2024-08-09T20:12:37.697+0200] {processor.py:161} INFO - Started process (PID=22028) to work on /home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-09T20:12:37.698+0200] {processor.py:830} INFO - Processing file /home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-08-09T20:12:37.699+0200] {logging_mixin.py:188} INFO - [2024-08-09T20:12:37.699+0200] {dagbag.py:545} INFO - Filling up the DagBag from /home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-09T20:12:37.700+0200] {logging_mixin.py:188} INFO - [2024-08-09T20:12:37.700+0200] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-08-09T20:12:37.700+0200] {logging_mixin.py:188} INFO - [2024-08-09T20:12:37.700+0200] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-08-09T20:12:37.701+0200] {processor.py:842} WARNING - No viable dags retrieved from /home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-09T20:12:37.706+0200] {processor.py:183} INFO - Processing /home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.012 seconds
[2024-08-09T20:14:02.203+0200] {processor.py:161} INFO - Started process (PID=22155) to work on /home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-09T20:14:02.204+0200] {processor.py:830} INFO - Processing file /home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-08-09T20:14:02.205+0200] {logging_mixin.py:188} INFO - [2024-08-09T20:14:02.204+0200] {dagbag.py:545} INFO - Filling up the DagBag from /home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-09T20:14:02.206+0200] {logging_mixin.py:188} INFO - [2024-08-09T20:14:02.206+0200] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-08-09T20:14:02.206+0200] {logging_mixin.py:188} INFO - [2024-08-09T20:14:02.206+0200] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-08-09T20:14:02.207+0200] {processor.py:842} WARNING - No viable dags retrieved from /home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-09T20:14:02.212+0200] {processor.py:183} INFO - Processing /home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.012 seconds
[2024-08-09T20:15:12.459+0200] {processor.py:161} INFO - Started process (PID=22268) to work on /home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-09T20:15:12.460+0200] {processor.py:830} INFO - Processing file /home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-08-09T20:15:12.460+0200] {logging_mixin.py:188} INFO - [2024-08-09T20:15:12.460+0200] {dagbag.py:545} INFO - Filling up the DagBag from /home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-09T20:15:12.462+0200] {logging_mixin.py:188} INFO - [2024-08-09T20:15:12.462+0200] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-08-09T20:15:12.462+0200] {logging_mixin.py:188} INFO - [2024-08-09T20:15:12.462+0200] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-08-09T20:15:12.462+0200] {processor.py:842} WARNING - No viable dags retrieved from /home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-09T20:15:12.468+0200] {processor.py:183} INFO - Processing /home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.012 seconds
[2024-08-09T20:16:24.203+0200] {processor.py:161} INFO - Started process (PID=22353) to work on /home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-09T20:16:24.204+0200] {processor.py:830} INFO - Processing file /home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-08-09T20:16:24.205+0200] {logging_mixin.py:188} INFO - [2024-08-09T20:16:24.205+0200] {dagbag.py:545} INFO - Filling up the DagBag from /home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-09T20:16:24.206+0200] {logging_mixin.py:188} INFO - [2024-08-09T20:16:24.206+0200] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-08-09T20:16:24.207+0200] {logging_mixin.py:188} INFO - [2024-08-09T20:16:24.207+0200] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-08-09T20:16:24.207+0200] {processor.py:842} WARNING - No viable dags retrieved from /home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-09T20:16:24.212+0200] {processor.py:183} INFO - Processing /home/bvpelt/Develop/airflowtut/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.012 seconds
